README.md

```diff
@@ -67,6 +67,29 @@ commands:
 
 Full installation instructions are available in [INSTALL.md](https://github.com/metabrainz/acousticbrainz-server/blob/master/INSTALL.md) file.
 
+## Login and Datasets
```

Comment:
I just realised that this is also duplicated here: 
https://github.com/metabrainz/acousticbrainz-server/blob/master/INSTALL.md#configuring-musicbrainz-oauth

We should work out where the split of these files should be. The steps that are common to both vagrant and manual install could be either in readme, or in the second half of install. Then whatever file they are not in can just say "see here for next steps"